packages/block-editor/src/components/block-preview/style.scss

```diff
@@ -36,3 +36,8 @@
 		}
 	}
 }
+
+// Hide inserter from previews.
+.block-editor-block-preview__content-iframe .block-list-appender {
```

Comment:
We already have a lot of similar styles above, why didn't we just nest this in `.block-editor-block-preview__container` above? instead of adding this new class?

Reply:
I guess the fact that we introduced iframes for previews is preventing all these resets from applying, in that case, maybe we should move all these reset styles inside the iframe selector. cc @ellatrix

Reply:
I'm happy to try a followup here. But from what I can tell, we actually do need both the outer and the inner container. The outer sets things like border, border radius, pointer cursor, all unscaled. The inner is only for content inside the iframe, and as you found, needs to be unscoped.

Reply:
Ideally we should be able to render a block list in a disabled mode that doesn't render any appenders. I thought we actually already did that because other appenders didn't show before in the preview.